.github/actions/setup-project/action.yml

```diff
@@ -62,7 +62,7 @@ runs:
     - name: Run build
       if: inputs.skip-build != 'true' && steps.cache-build.outputs.cache-hit != 'true'
       shell: bash
-      run: yarn build && yarn build:umd
+      run: yarn build && yarn build:umd && yarn clean:exports
```

Comment:
This seems to be a regression that was introduced, this script was previously ran on CI and was lost in refactoring. I've restored it here, and I will create a patch to fix this for `v5` as well.

Reply:
I believe this removal was actually intentional, as we now lo longer export our demo files in the top level index of react-core, which is what that `clean:exports` was fixing before.